config/rbac/role_custom.yaml

```diff
@@ -1,94 +0,0 @@
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  creationTimestamp: null
-  name: manager-role
-rules:
-- apiGroups:
-  - ""
-  resources:
-  - serviceaccounts
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - "rbac.authorization.k8s.io"
-  resources:
-  - roles
-  - rolebindings
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - build.kairos.io
-  resources:
-  - osartifacts
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - build.kairos.io
-  resources:
-  - osartifacts/finalizers
-  verbs:
-  - update
-- apiGroups:
-  - build.kairos.io
-  resources:
-  - osartifacts/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - build.kairos.io
-  resources:
-  - osartifacts/finalizers
-  verbs:
-  - update
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  verbs:
-  - get
-  - create
-  - update
```

Comment:
This part is still needed. I don't know about the rest here but this one makes it fail when I apply the OsArtifact in the PR description.

Reply:
I think this was resolved with the latest changes, feel free to unresolve if not.